lib/config/options/index.ts

```diff
@@ -1807,7 +1807,7 @@ const options: RenovateOptions[] = [
   {
     name: 'prBodyTemplate',
```

Comment:
Not sure if this is a `docs:` or `chore:` type of change, please edit the PR title if needed. 😉